pygmt/datasets/samples.py

```diff
@@ -79,6 +78,7 @@ def load_sample_data(name):
 
     # Dictionary of private load functions
     load_func = {
+        "bathymetry": _load_sample_bathymetry,
```

Comment:
The name `_load_sample_bathymetry` is a little too general. Perhaps we should rename it to something like `_load_baja_california_bathymetry`. 

This function is used internally, so the renaming doesn't cause incompatible changes.